Dockerfile

```diff
@@ -6,8 +6,11 @@ RUN apk add --update-cache py3-pip ca-certificates py3-certifi py3-lxml\
                            jpeg-dev zlib-dev freetype-dev lcms2-dev openjpeg-dev \
                            tiff-dev tk-dev tcl-dev
 
-ADD . /build/
-RUN pip3 install -e /build/[u2f]
+COPY setup.py README.rst requirements.txt /build/
+RUN pip3 install -r /build/requirements.txt
+
+COPY aws_google_auth /build/aws_google_auth
+RUN pip3 install -e /build
```

Comment:
Please update to the [u2f] build version.

Reply:
Yep the removal of `[u2f]` wasn't intentional :)